Makefile

```diff
@@ -1,3 +1,8 @@
+PWD=$(shell pwd)
+export C_INCLUDE_PATH=${PWD}/extern/tmp
+export LIBRARY_PATH=${PWD}/extern/tmp
+export LD_LIBRARY_PATH=${PWD}/extern/tmp
+
```

Comment:
What I mean is that if user already installs rocksdb, we can use it, not build again.